bamboo-storage/src/main/java/com/pushtorefresh/android/bamboostorage/operation/get/PreparedGetCursor.java

```diff
@@ -42,6 +46,32 @@
         });
     }
 
+    @NonNull @Override public Observable<Cursor> createObservableStream() {
+        final Set<String> tables;
+
+        if (query != null) {
+            tables = new HashSet<>(1);
```

Comment:
Just memory optimization: Regular `Query` can work with only one table (see next line), while `RawQuery` can work with multiple tables (join and so on)